### DISCUSSION TITLE
Kconfig warnign

### DISCUSSION THREAD

**Discussion by ngocuong86** [zephyrproject-rtos/zephyr#56651] [Q&A]
Hi everyone.
I use: 
west version: 1.0.0
cmake verssion: 3.26.2
zephyr version 0.15
But I tried build example lighting-app have this error.
`cuong@cuong-Aspire-A715-72G:~/connectedhomeip/examples/lighting-app/telink$ west build
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base (cached)).
-- Application: /home/cuong/connectedhomeip/examples/lighting-app/telink
-- Cache files will be written to: /home/cuong/.cache/zephyr
-- Zephyr version: 3.2.99 (/home/cuong/zephyrproject/zephyr)
-- Found west (found suitable version "0.12.0", minimum required is "0.7.1")
-- Board: tlsr9518adk80d
-- Found host-tools: zephyr 0.15.0 (/home/cuong/zephyr-sdk-0.15.0)
-- Found toolchain: zephyr 0.15.0 (/home/cuong/zephyr-sdk-0.15.0)
-- Found BOARD.dts: /home/cuong/zephyrproject/zephyr/boards/riscv/tlsr9518adk80d/tlsr9518adk80d.dts
-- Generated zephyr.dts: /home/cuong/connectedhomeip/examples/lighting-app/telink/build/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /home/cuong/connectedhomeip/examples/lighting-app/telink/build/zephyr/include/generated/devicetree_generated.h
-- Including generated dts.cmake file: /home/cuong/connectedhomeip/examples/lighting-app/telink/build/zephyr/dts.cmake
Parsing /home/cuong/zephyrproject/zephyr/Kconfig
Loaded configuration '/home/cuong/zephyrproject/zephyr/boards/riscv/tlsr9518adk80d/tlsr9518adk80d_defconfig'
Merged configuration '/home/cuong/connectedhomeip/config/telink/app/zephyr.conf'
Merged configuration '/home/cuong/connectedhomeip/examples/lighting-app/telink/prj.conf'

warning: SHELL_BACKEND_SERIAL_RX_RING_BUFFER_SIZE (defined at
subsys/shell/backends/Kconfig.backends:58) was assigned the value '255' but got the value ''. Check
these unsatisfied dependencies: SHELL_BACKEND_SERIAL (=n), SHELL_BACKENDS (=n), SHELL (=n). See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_SHELL_BACKEND_SERIAL_RX_RING_BUFFER_SIZE
and/or look up SHELL_BACKEND_SERIAL_RX_RING_BUFFER_SIZE in the menuconfig/guiconfig interface. The
Application Development Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices
sections of the manual might be helpful too.

warning: The choice symbol MCUBOOT_UTIL_LOG_LEVEL_INF (defined at
subsys/logging/Kconfig.template.log_config:17) was selected (set =y), but no symbol ended up as the
choice selection. See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_MCUBOOT_UTIL_LOG_LEVEL_INF and/or look up
MCUBOOT_UTIL_LOG_LEVEL_INF in the menuconfig/guiconfig interface. The Application Development
Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual
might be helpful too.

warning: The choice symbol OPENTHREAD_L2_LOG_LEVEL_INF (defined at
subsys/net/Kconfig.template.log_config.net:31) was selected (set =y), but no symbol ended up as the
choice selection. See
http://docs.zephyrproject.org/latest/kconfig.html#CONFIG_OPENTHREAD_L2_LOG_LEVEL_INF and/or look up
OPENTHREAD_L2_LOG_LEVEL_INF in the menuconfig/guiconfig interface. The Application Development
Primer, Setting Configuration Values, and Kconfig - Tips and Best Practices sections of the manual
might be helpful too.

/home/cuong/connectedhomeip/config/telink/app/zephyr.conf:86: warning: attempt to assign the value '"IEEE802154_b91"' to the undefined symbol NET_CONFIG_IEEE802154_DEV_NAME

/home/cuong/connectedhomeip/examples/lighting-app/telink/prj.conf:36: warning: attempt to assign the value '"IEEE802154_b91"' to the undefined symbol NET_CONFIG_IEEE802154_DEV_NAME

error: Aborting due to Kconfig warnings

CMake Error at /home/cuong/zephyrproject/zephyr/cmake/modules/kconfig.cmake:337 (message):
  command failed with return code: 1
Call Stack (most recent call first):
  /home/cuong/zephyrproject/zephyr/cmake/modules/zephyr_default.cmake:108 (include)
  /home/cuong/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:66 (include)
  /home/cuong/zephyrproject/zephyr/share/zephyr-package/cmake/ZephyrConfig.cmake:97 (include_boilerplate)
  CMakeLists.txt:29 (find_package)

-- Configuring incomplete, errors occurred!
FATAL ERROR: command exited with status 1: /usr/bin/cmake -DWEST_PYTHON=/usr/bin/python3 -B/home/cuong/connectedhomeip/examples/lighting-app/telink/build -GNinja -S/home/cuong/connectedhomeip/examples/lighting-app/telink
`
HOW TO FIX IT. THANKS

**Comment by yashi** [comment]
I think it's better to ask [over there](https://github.com/project-chip/connectedhomeip).
